test.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
What is the purpose of this file? It seems that we can delete it since it's not used anywhere.

Reply:
I think i needed it to test restricting the build matrix but ended up going with a different solution. I'll remove it